list

```diff
@@ -1,1181 +1,283 @@
 1
-13
-challenge-map-set.wad
-Challenge Maps Set
-A set of king_of_nowhere’s challenge maps, curated by Nordfriese. Contained maps: Dust in the Wind, To Make the Desert Bloom, The Great Escape, Concentric Rings. Recommended for advanced players who find playing alone with Poor Hamlet against six or more teamed-up trading outpost AIs too easy.
-king_of_nowhere
+14
+legacy_ui.wad
+Widelands legacy
+The old Widelands beta UI.
+matthiakl
 Nordfriese
```

Comment:
```suggestion
matthiakl
```
The uploader name is hardcoded because until now I was the only one who uploaded stuff

Reply:
This could need a hint in the Readme :)

Reply:
I'll add one. LGTM :)